test/rpc/test_rpc_call.py

```diff
@@ -93,6 +105,7 @@ def test_call_raises_on_invalid_calldata(deploy_info):
 
 # This test will fail since we are throwing a custom error block_hash different from `latest`
 @pytest.mark.xfail
```

Comment:
True.

I guess it was supposed to be maybe a temporary change so the test would be viable after removing it.

But yeah, I just realized how dangerous xfails are (since any fail reason will be accepted).